File: core/core-web/src/test/java/ch/difty/scipamato/core/web/resources/jasper/PaperSummaryTableReportResourceReferenceTest.java
package ch.difty.scipamato.core.web.resources.jasper;

class PaperSummaryTableReportResourceReferenceTest
    extends JasperReportResourceReferenceTest<PaperSummaryTableReportResourceReference> {

    @Override
    protected PaperSummaryTableReportResourceReference getResourceReference() {
        return PaperSummaryTableReportResourceReference.get();
    }

    @Override
    protected String getReportBaseName() {
        return "paper_summary_table_A4";
    }

    @Override
    protected String getResourceReferencePath() {
        return "ch.difty.scipamato.core.web.resources.jasper.PaperSummaryTableReportResourceReference";
    }
}

File: core/core-web/src/test/kotlin/ch/difty/scipamato/core/web/resources/jasper/PaperSummaryTableReportResourceReferenceTest.kt
package ch.difty.scipamato.core.web.resources.jasper

internal class PaperSummaryTableReportResourceReferenceTest :
    JasperReportResourceReferenceTest<PaperSummaryTableReportResourceReference>() {

    override val resourceReference: PaperSummaryTableReportResourceReference
        get() = PaperSummaryTableReportResourceReference.get()

    override val reportBaseName: String
        get() = "paper_summary_table_A4"

    override val resourceReferencePath: String
        get() = "ch.difty.scipamato.core.web.resources.jasper.PaperSummaryTableReportResourceReference"
}
